### DISCUSSION TITLE
是否样式提示可以更丰富？

### DISCUSSION THREAD

**Discussion by lishaobos** [unocss/unocss#1834] [Ideas]
# 问题

当我为某个元素写下很多个 uno class 的时候，我很难一眼识别出我这个元素最终的样式集合究竟是什么，在做一些样式调整的时候感觉没有那么方便。

![image](https://user-images.githubusercontent.com/46661044/199452238-cf80c335-e69f-49a5-8eb3-3e98851091e5.png)

# 丰富 vscode 插件

我自己修改了 unocss 的 vscode 插件，当我选中某些 unocss 片段的时候，触摸 emoji，直接展示选中样式的集合。

![image](https://user-images.githubusercontent.com/46661044/199452761-73a6a1e6-2515-4307-a638-d43e39c435a0.png)

# 期望

这个是否可以加入 unocss 的 vscode 插件功能

**Comment by antfu** [comment]
Sounds like a good idea. PR welcome :)

**Reply by lishaobos** [reply]
ok
